src/scopes.py

```diff
@@ -16,15 +19,16 @@ def changeX():
 # This nested function has a similar problem.
 
 def outer():
-    y = 120
+    outer.y = 120
```

Comment:
This works, but there's also a way to do it with the `nonlocal` keyword like you used `global` above https://www.w3schools.com/python/ref_keyword_nonlocal.asp